gradle/maven.gradle

```diff
@@ -1,5 +1,5 @@
 allprojects {
-  version = "5.2.8-SNAPSHOT"
+  version = "5.3.0"
```

Comment:
Keep `SNAPSHOT` in the version number. That prevents developers from accidentally publishing a version.

Reply:
Although once a version is published, it could not be overridden afaik but we can keep SNAPSHOT to be on safer side.